Tools/TestWebKitAPI/Tests/WTF/cocoa/TextStreamCocoa.cpp

```diff
@@ -34,3 +34,10 @@ TEST(WTF_TextStream, CFString)
     ts << reinterpret_cast<id>(const_cast<CFMutableStringRef>(CFSTR("Test")));
     EXPECT_EQ(ts.release(), "Test"_s);
 }
+
+TEST(WTF_TextStream, Hex)
+{
+    TextStream ts;
+    ts << hex(18);
```

Comment:
might as well use something that produces a-f so we can test the case of the output.